commitlint.config.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   rules: {
     'scope-enum': [2, 'always', [
       'React', 'React Native', 'SDK',
-      'DataTable'
+      'DataTable', 'SCR', 'Docs'
```

Comment:
I suggest removing `Docs` and using `docs` type instead of `chore(Docs)`

Reply:
Also I'm not sure we should introduce SCR scope as it's completely internal stuff